css/global.css

```diff
@@ -724,9 +813,104 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 }
 
 /* Tablet and smaller 768px */
-@media screen and (max-width: var(--mq-width-tablet-max)) {
+@media screen and (max-width: 768px) {
+	/* 
+	===================================
+	Hero Adjustment
+	===================================
+	*/
+
+	.hero {
+		/* layout for <img> and wrapper */
+		flex-direction: column;
+
+		color: var(--color-onyx);
+		padding-top: 0;
+
+		gap: var(--spacing-lg);
+	}
+
+	/* revealing mobile hero img */
+	.hero-img-mobile {
+		display: block;
+		visibility: visible;
+	}
+
+	/* 
+	===================================
+	Modal Adjustment
+	===================================
+	*/
+	.modal-content {
+		width: 90%;
+		padding: var(--spacing-sm);
+	}
 }
 
-/* Mobile Only 400px */
-@media screen and (max-width: var(--mq-width-phone-max)) {
+/* custom breakpoint 470px */
+@media screen and (max-width: 470px) {
+	/* 
+	===================================
+	Mobile Text
+	===================================
+	*/
+
+	.hero h2 {
+		font-size: var(--font-size-title-med);
+	}
+	body {
+		font-size: var(--font-size-body-reg);
+	}
+	/* 
+	===================================
+	Footer
+	===================================
+	*/
+
+	.column-container {
+		flex-direction: column;
+		justify-content: flex-start;
+		align-items: center;
+		gap: var(--spacing-med);
+	}
+
+	footer .company,
+	footer nav ul {
+		align-items: center;
+	}
+	/* newsletter form */
+	.newsletter-form {
+		text-align: center;
+	}
+
+	.newsletter-form p {
+		width: 100%;
+	}
+
+	.newsletter-form .input-label-container {
+		flex-direction: column;
+		align-items: center;
+	}
+
+	.newsletter-form input[type="email"] {
+		width: 100%;
+	}
+
+	/* 
+	===================================
+	Modal 
+	===================================
+	*/
+
+	.modal-text-container h4 {
+		font-size: var(--font-size-title-sm);
+	}
 }
+
+/* NOT CURRENTLY WORKING */
```

Comment:
When I was working with the Variables Team, we discovered variables don't work in media queries like this (unless you are using Sass or other compiler). 

Here's the post Roo found: https://bholmes.dev/blog/alternative-to-css-variable-media-queries/